### PR TITLE
html-forms//MichaelCardona

### PR DESCRIPTION
**Pull request by MichaelCardona**
Screenshot 2021-03-11 211433

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form</h1>
+
+  <form action="index.html" method="GET">
+    <div>
+      <label for="Email">Email: </label> <br/>
+        <input type="text">
```
Comment (BrettAlbright) [inline html-forms/index.html]: missing id attribute

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    </div>
+
+    <div>
+    <label for="why is coding cool?">Why is coding cool?</label> <br/>
+      <select name="reasons">
```
Comment (BrettAlbright) [inline html-forms/index.html]: missing id attribute

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    </div>
+
+    <div>
+      <label for="question">Would you like to sign up for more information?</label><br/>
+      <input type="checkbox" name="Sign up" value="Sign me up" /> Sign me up!
+    </div>
+
+    <div>
+      <label for="comments">Comments</label> <br/>
+      <textarea name="Comments" cols="20" rows="4"></textarea>
```
Comment (BrettAlbright) [inline html-forms/index.html]: missing id attribute

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    </div>
+  </form>
```
Comment (BrettAlbright) [inline html-forms/index.html:42]: you also need a submit input to submit the form

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
some updates needed

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**